astroid/exceptions.py

```diff
@@ -114,6 +114,8 @@ def __init__(
 class AstroidSyntaxError(AstroidBuildingError):
     """Exception class used when a module can't be parsed."""
 
+    # this should probably be a SyntaxError subclass.
```

Comment:
We also re-raise it from `TypeError`, `ValueError` and `LookupError`. So it doesn't seem to only be a `SyntaxError`?

Reply:
I don't know, this is out of the scope if this PR anyway.

Reply:
Wouldn't the comment then not also be out of scope?